api/src/main/resources/public/api/v0/schema/jobspec_v0.schema.json

```diff
@@ -79,6 +79,38 @@
           "description": "How much disk space is needed for this job. This number does not have to be an integer, but can be a fraction.",
           "minimum": 0
         },
+        "ucr": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "image": {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "id": {
+                  "type": "string",
+                  "documentation": "The repository image name."
+                },
+                "kind": {
+                  "type": "string",
+                  "documentation": "The type of image",
+                  "enum": ["docker", "appc"],
+                  "default": "docker"
+                }
+              },
+              "required": ["id"]
+            },
+            "forcePullImage": {
```

Comment:
I think this should be in image

```
"image": {
  "id": "...",
  "kind": "docker",
  "forcePull": false
}

Reply:
https://github.com/mesosphere/marathon/blob/master/docs/docs/rest-api/public/api/v2/types/podContainer.raml#L32

Reply:
good catch... I was going mainly off what was put in the doc... but clearly that was incorrect.